packages/components/select-v2/src/useSelect.ts

```diff
@@ -151,9 +152,11 @@ const useSelect = (props: ISelectV2Props, emit) => {
   )
 
   const validateState = computed(() => elFormItem?.validateState || '')
-  const validateIcon = computed(
-    () => ValidateComponentsMap[validateState.value]
-  )
+  // eslint-disable-next-line vue/return-in-computed-property
+  const validateIcon = computed(() => {
+    if (!validateState.value) return
+    return ValidateComponentsMap[validateState.value]
```

Comment:
There seems to be no need to add additional judgment here. If `validateState.value` does not exist, the returned value should also be `undefined`.

Reply:
I see what you mean, however,  `Typescript` will report “Element implicitly has an 'any' type” for it.

Reply:
Should we add a comment here about this kind of writing?

Reply:
Agree, will do.